docs/tutorial/using-grpc-plugins.mdx

```diff
@@ -0,0 +1,552 @@
+---
+title: "Using Cosmo Cloud gRPC plugins to quickly build and deploy a gRPC subgraph"
+description: "Learn how to use gRPC plugins with Cosmo Cloud & Cosmo Router"
+icon: plug
+---
+
+## Prerequisites
+
+- [Cosmo CLI](/cli/intro) installed and configured
```

Comment:
I think, specifying the minimum required version would be helpful

Reply:
It's not released yet unfortunately

Reply:
I added a big TODO to add these when we release